### DISCUSSION TITLE
Sample Opaque in 1.0.0.beta?

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#238] [Q&A]
How does the sampling of the opaque layer work now in 1.0.0.beta? The old shader does not work anymore and I have tried to fix it but it didnt work at all (mostly because I dont really understand what's going on). How would I need to change the `sample_opaque` function to get it working again?

https://github.com/bnpr/Malt/discussions/179#discussioncomment-1867947

**Comment by pragma37** [comment]
Seems like this (hidden) feature slipped through the net in the update.
I'll update this answer once I update the code to support it again.

**Comment by pragma37** [comment]
I added back this feature in a slightly different way.
The Plugins folder has a new plugin (Experimental) that, for now, only adds a new OpaqueLayer node:
![imagen](https://user-images.githubusercontent.com/36610999/153455217-51433ae3-c9a4-4045-bc5f-84c85099390f.png)
It can read any of the RenderLayer outputs, referenced by name.
In the future, this will likely be replaced by a LastLayer (including transparent layers) node in the main addon.
